imgui/src/io.rs

```diff
@@ -458,6 +458,12 @@ impl Io {
         }
     }
 
+    pub fn add_focus_event(&mut self, focused: bool) {
```

Comment:
Can you add documentation above this?

I would start by ripping out the documentation from `ImGuiIo_AddFocusEvent` itself in the C++ (I'll post that below):

```
// Queue a gain/loss of focus for the application (generally based on OS/platform focus of your window)
```

I'd also add a note re: `ConfigDebugIgnoreFocusLoss`, which basically makes imgui ignore this event firing.

Lastly, I'd add a doc alias `#[doc(alias = "AddFocusEvent")]` for the C++ api to bind to us directly.